clouddrift/datasets.py

```diff
@@ -232,6 +233,53 @@ def glad(decode_times: bool = True) -> xr.Dataset:
     return ds
 
 
+def hurdat2(option: BasinOption = BasinOption.BOTH):
+    """Returns the hurricane (Atlantic Basin) v2 (HURDAT2) dataset as a ragged array
+    Xarray dataset.
+
+    The function will first look for the ragged-array dataset on the local
+    filesystem. If it is not found, the dataset will be downloaded using the
+    corresponding adapter function and stored for later access.
+
+    The upstream data is available at https://www.aoml.noaa.gov/hrd/hurdat/hurdat2.html
+
+    Parameters
+    ----------
+    option : BasinOption, default: BOTH
```

Comment:
how does one use this? `ds = hurdat2(option=1)`? Here list what the option should be 1,2,3? Or "Atlantic", "Pacific","Both" (Default)?